### PR TITLE
Add author to responses and get author endpoint

### PR DESCRIPTION
**Pull request by nipisarev**

### CODE REVIEW

**Review by abelharisov — APPROVED** [review]

File: paths/authors.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  summary: Get all authors.
+  description: |
+    This method allows to retrieve all authors in selected project.
+  parameters:
+    - $ref: '../parameters/filter/Authors.yaml'
+    - $ref: '../parameters/pagination/Limit.yaml'
+    - $ref: '../parameters/pagination/Offset.yaml'
+  responses:
+    '200':
+      description: An author.
```
Comment (sklmx) [inline paths/authors.yaml:13]: Author list

File: schemas/responses/Author.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+allOf:
+  - $ref: './Base.yaml'
+  - type: object
+    properties:
+      result:
+        $ref: '../Author.yaml'
```
Comment (sklmx) [inline schemas/responses/Author.yaml:6]: add new line, please

**Review by sklmx — CHANGES_REQUESTED** [review]

File: api.yaml [hunk 3]
```diff
@@ -2327,6 +2397,9 @@ components:
         member_id:
```
Comment (sklmx) [inline api.yaml:2397]: set it as deprecated

Comment (nipisarev) [reply api.yaml:2397]: fixed

**Pull request merged**